docs/csharp/toc.yml

```
@@ -1620,13 +1621,13 @@
       displayName: conditional, ternary, ref
     - name: "! (null-forgiving) operator"
       href: language-reference/operators/null-forgiving.md
-      displayName: nullable reference, null-forgiveness
+      displayName: nullable reference, null-forgiveness, damn-it
```

Please let me know if that addition is not fine.

---

Thanks for calling this one out. We should remove it, as it doesn't pass the profanity guidelines
```suggestion
      displayName: nullable reference, null-forgiveness
```